doc/lua_api.txt

```diff
@@ -6432,6 +6432,25 @@ Doing so is much less error-prone and you will never need to wonder if the
 object you are working with still exists.
 
 
+### Attachment
+
+It is possible to attach objects to other objects (`set_attach` method).
+
+When an object is attached, its position gets relative to the parent`s one and
+rotation is a sum of its own, the parent`s and relative rotations. Standard
+methods like 'set_pos' and 'set_rotation' won`t work, you should call repeatedly
+`set_attach` with already new position or rotation tables.
+
+**Bug**: keep in mind to multiply relative position by 10 when attaching to make it actually proper!
+
+Example:
+`child:set_attach(parent, "", vector.multiply(vector.subtract(child:get_pos(), parent:get_pos()), 10))`.
+
+It is also possible to set bones for parent objects like in Blender (`set_bone_position` method)
+and use them for child ones to adjust their position/rotation. After assignment a bone to an attached object,
+it will follow movement and rotation of that bone.
```

Comment:
```suggestion
### Attachments

It is possible to attach objects to other objects (`set_attach` method).

When an object is attached, it is positioned relative to the parent's position and rotation.
`get_pos` and `get_rotation` will always return the parent's values and can't be set directly.
To change position or rotation call `set_attach` again with the new values.

**Note**: Just like model dimensions, the relative position in `set_attach` must be multiplied by 10.

It is also possible to attach to a bone of the parent object (as defined in the model).
After assignment a bone to an attached object, the child will follow movement and rotation of that bone.
```

Reply:
I think it is necessary to mention also the child rotation is a sum of its own, the parent's snd relative rotations, at least I checked it experimentally.